src/semra/client.py

```diff
@@ -480,7 +570,7 @@ def get_connected_component(
             WHERE a <> b
             AND a.curie in $curies AND b.curie in $curies
             // Make sure the evidence is not an inversion of chaining
-            AND NOT (e.mapping_justification IN ['{CHAIN_MAPPING.curie}', '{INVERSION_MAPPING.curie}'])
+            AND NOT (e.mapping_justification IN ['`{CHAIN_MAPPING.curie}`', '`{INVERSION_MAPPING.curie}`'])
```

Comment:
In this context I don't think the backticks are needed but it's fine to have them.

Reply:
Actually, the backticks are incorrect here and mess up the result, we need to remove them.
```suggestion
            AND NOT (e.mapping_justification IN ['{CHAIN_MAPPING.curie}', '{INVERSION_MAPPING.curie}'])
```